crates/holochain_cascade/src/lib.rs

```diff
@@ -422,6 +438,37 @@ impl CascadeImpl {
         Ok(())
     }
 
+    /// Fetch a ChainOp from the network by ActionHash and ChainOpType
+    #[cfg_attr(feature = "instrument", tracing::instrument(skip(self)))]
+    pub async fn fetch_op_by_type(
```

Comment:
This deviates from the pattern to return a unit value. I find it important to integration test this function, so I added the return value.